src/main/java/org/jenkinsci/plugins/durabletask/BourneShellScript.java

```diff
@@ -107,7 +140,22 @@ public String getScript() {
         if (script.isEmpty()) {
             listener.getLogger().println("Warning: was asked to run an empty script");
         }
-        OsType os = ws.act(new getOsType());
+
+        final Jenkins jenkins = Jenkins.get();
+
+        Manifest manifest;
+        try (InputStream manifestStream = DurableTask.class.getResourceAsStream("/META-INF/MANIFEST.MF")) {
```

Comment:
Rather than working with `Manifest` directly I would maybe do something like `Jenkins.get().getPluginManger().getPlugin("durable-task").getVersion()`.

Reply:
had no idea about that! yes, that helps immensely!

Reply:
Done in 5f7d04e.